tests/phpstan/bootstrap.php

```diff
@@ -0,0 +1,100 @@
+<?php
+// phpcs:ignoreFile
```

Comment:
This file should be added to the `phpcs.xml.dist` file for ignoring instead of having an inline comment.